portafly/src/i18n/locales/en/audience/account/account.yml

```diff
@@ -0,0 +1,67 @@
+---
+# Accounts > Account overview
+
+page_title: Accounts
+page_title_description: Page title of the subsection for Accounts Overview
+
+# Top area
+
+body_title: "Account: {{organization_name}}"
+body_title_description: Body title of the page for Accounts Overview
+
+# Tabs
+
+overview_title: Overview
+overview_description: This is the title of the tab containing the organization or group name, the administrator name/email address, signup date, and state.
+applications_title: Applications
+applications_description: This is the title of the tab containing the list of applications on the account, their names, the associated product, its plan, creation date, and its current state.
+
+# Main area
+# Tab overview
+
+organization_label: Organization/Group Name
+organization_value: {{organization_name}}
+administrator_label: Administrator
+administrator_value: {{admin_name (name@emailaddress.com)}}
+signup_label: Signed up on
+sigup_value: {{YYYY-MM-DD, 00:00}}
+state_label: State
+state_value: {{application_status}}
+
+# Tab Applications
+# Top area
+
+create_application_button: Create application
+create_application_button_description: Button to create an application
+create_application_button_aria_label: Create application
+create_application_button_aria_label_description: Aria label for create application button
+
+applications_filter: Filter by name
+applications_filter_description: Dropdown list to filter applications by specified option
+applications_filter_options:
+  by_name: Name
+  by_product: Product
+  by_plan: Plan
+  by_created_on: Created on
+  by_state: State
+applications_filter_field: "Filter by {{filter_option}}"
+applications_filter_field_description: Filter by typeahead a text input.
+applications_filter_field_aria_label: Filter text input
+applications_filter_field_aria_label_description: Aria label for the filter text input
+
+## Header
+applications_table:
+ aria_label: Applications table
+ aria_label_description: Aria label of the table of applications
+ empty_state: Currently, there are no applications to show. Add applications.
```

Comment:
@porueesq @dfennessy can we mark all strings for empty states as temporary until mockups for empty states are released?

Reply:
@cntlsn What do you suggest to highlight this temporary state: a comment line, for instance?

Reply:
@porueesq if inline comments work I think that would be the best option. something like:
`empty_state: Currently, there are no applications to show. Add applications.    # Temporary`

Reply:
@josemigallas do inline comments work for you?

Reply:
Yeppers 👍